tests/agents/json-specs/cgroup_parsing.json

```diff
@@ -0,0 +1,22 @@
+{
+	"testUnderscores": {
```

Comment:
[minor] other json files are indented with two spaces per indent level.

Reply:
Well, it looks like most of them are 😄 - `w3c_distributed_tracing.json` uses tabs as well.
Let's go with the majority though 👍 .

Reply:
I vaguely remember that this W3C specification JSON comes from an external repository, and keeping it as-is would have helped doing diffs with the original source in case it would have been updated.